### ISSUE TITLE
Unnecessary aria-expanded on content div in Collapse

### ISSUE THREAD

**Issue by kalongthewires** [reactstrap/reactstrap#1757]
- components: `Collapse`
- reactstrap version `#8.2.0`
- import method `es`
- react version `16.12.0`
- bootstrap version `#4.3.1`

### What is happening?

The `aria-expanded` attribute is included on the collapse content div.

![image](https://user-images.githubusercontent.com/2043335/72555572-d2a5ee00-386a-11ea-8140-6cc87ca3bb79.png)

### What should be happening?

No `aria-expanded` attribute should be present on the div. Per the WAI-ARIA documentation, `aria-expanded` is set on the button in the header only (https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html).

The div `aria-expanded` attribute seems to have been introduced in this PR: https://github.com/reactstrap/reactstrap/pull/1657/files. Rolling back these changes should resolve the issue.

### Steps to reproduce issue

1. View the code generated by the Collapse component.

### Error message in console

N/A

### Code

N/A
